src/services/component-service.ts

```diff
@@ -64,27 +67,30 @@ export class ComponentService {
     }))
   }
 
+  public static getComponentPath(component: Component<ComponentType>): string {
+    const { name, type } = component
+    const componentPath =
+      type === ComponentType.MICROSERVICE
+        ? path.resolve(MICROSERVICES_FOLDER, name)
+        : path.resolve(MICROFRONTENDS_FOLDER, name)
+    return componentPath
+  }
+
   public async build(name: string): Promise<ProcessExecutionResult> {
```

Comment:
Is it presumed that there would be no duplicate names for a micro frontend and microservice? @cecchisandrone

Reply:
good point. @alepintus can you check this and enforce it at the moment of MFE/MS add command?

Reply:
done